core/app/models/workarea/navigation/taxon.rb

```diff
@@ -126,6 +126,10 @@ def search_results?
         navigable.is_a?(SearchResults)
       end
 
+      def show_in_sitemap?
```

Comment:
So now that we have this method, it feels like the HTML sitemap should also respect this as well as the XML one. Can you update that?